src/line_opacity.jl

```diff
@@ -76,12 +74,116 @@ function line_absorption(linelist, λs, temp, nₑ, n_densities::Dict, partition
         end
 
         invΔλ_D = 1/Δλ_D
-        @inbounds view(α_lines, lb:ub) .+= line_profile.(line.wl, invΔλ_D, Δλ_L, line_amplitude,
+        @inbounds view(α_lines, lb:ub) .+= line_profile.(line.wl, invΔλ_D, Δλ_L, amplitude,
                                                          view(λs, lb:ub))
     end
     α_lines
 end
 
+"""
+    load_hydrogen_stark_profiles()
+
+Load the (doppler-convolved) Stark-broadening profiles from disk.
+"""
+function setup_hydrogen_stark_profiles(fname=joinpath(_data_dir, 
+                                                      "Stehle-Hutchson-hydrogen-profiles.h5"))
+    hline_stark_profiles = h5open(fname, "r") do fid
+        map(keys(fid)) do transition
+            temps = read(fid[transition], "temps")
+            nes = read(fid[transition], "electron_number_densities")
+            delta_nu_over_F0 = read(fid[transition], "delta_nu_over_F0")
+            
+            siz = length(delta_nu_over_F0)
+            nans = @SVector fill(NaN, siz)
+            P = read(fid[transition], "profile")
+            
+            (temps=temps, 
+             electron_number_densities=nes,
+             #flat BCs to allow wls very close to the line center
+             profile=LinearInterpolation((temps, nes, [floatmin() ; log.(delta_nu_over_F0[2:end])]), 
+                                         log.(P); extrapolation_bc=Flat()),
+             lower = HDF5.read_attribute(fid[transition], "lower"),
+             upper = HDF5.read_attribute(fid[transition], "upper"),
+             Kalpha = HDF5.read_attribute(fid[transition], "Kalpha"),
+             log_gf = HDF5.read_attribute(fid[transition], "log_gf"),
+             λ0 = LinearInterpolation((temps, nes), read(fid[transition], "lambda0") * 1e-8) 
+            )
+        end
+    end
+end
+
+
+#used in hydrogen_line_absorption
+_zero2epsilon(x) = x == 0 ? floatmin() : x
```

Comment:
```suggestion
_zero2epsilon(x) = x + (x == 0) * floatmin()
```
This is just a micro-optimization to explicitly avoid branches. Feel free to ignore it.